src/eip4824.ts

```diff
@@ -0,0 +1,155 @@
+import express from 'express';
+import { getSpace } from './helpers/spaces';
+import db, { sequencerDB } from './helpers/mysql';
+
+const router = express.Router();
+
+router.get('/:space', async (req, res) => {
+  let space: any = {};
+  const baseUrl = `${req.protocol}://${req.get('host')}${req.originalUrl}`;
+
+  try {
+    space = await getSpace(req.params.space);
+
+    if (!space.verified) return res.status(404).json({ error: 'INVALID' });
+  } catch (e) {
+    console.log(e);
+    return res.status(500).json({ error: 'NOT_FOUND' });
```

Comment:
May be 404 for `NOT_FOUND` and 400 for  `INVALID`
```suggestion
    if (!space.verified) return res.status(400).json({ error: 'INVALID' });
  } catch (e) {
    console.log(e);
    return res.status(404).json({ error: 'NOT_FOUND' });
```